exocom-server/src/client-registry/client-registry.ls

```diff
@@ -38,19 +38,20 @@ class ClientRegistry
 
 
 
-  # Adds service routing configurations to the given setup
-  register-client: (service) ->
-    @clients[service.name] =
-      client-name: service.name
-      service-type: service.name
-      internal-namespace: @routing[service.name].internal-namespace
+  # registers the given service instance that just came online
+  register-client: (client) ->
+    @clients[client.name] =
+      client-name: client.name
```

Comment:
Do we need a step here to make sure that the client name is available, and to alter it if it isn't? In #26 you mention that the client names could be essentially the `service-type` followed by an incremental number, e.g. `web1`, `web2`.